README.md

```diff
@@ -52,32 +52,12 @@ browser**. To use this executable, simply type in a regular terminal:
 
 ### Installation on a local computer
 
-#### Installation with conda (recommended)
-
-In order to install `openPMD-viewer` with `conda`, please install the [Anaconda
-distribution](https://docs.anaconda.com/anaconda/install/), and then type
-```
-conda install -c rlehe openpmd_viewer

```

Comment:
you could actually keep this in here and merge it as soon as our feedstock is open.
The new line will be:
```
conda install -c conda-forge openpmd-viewer
```

Reply:
OK, sounds good. 
But it looks I should also wait for the conda-forge package to be updated for openPMD-viewer 1.0.0 first, correct?